config.guess

```diff
@@ -0,0 +1,1462 @@
+#! /bin/sh
```

Comment:
Why is this in the repo?

Reply:
are these files automatically generated?

Reply:
This one doesn't say it's automatically generated, but I don't see a reason why you'd copy and paste this full file into the repo so I assume this was generated by a tool you used.